hack/mirror-origin-tests.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+. $(dirname "$0")/origin-tests-common.sh
+
+skopeo_workdir=$(mktemp -d)
+mapping_file=$(mktemp)
+function finish {
+  rm -rf ${skopeo_workdir}
+  rm -f ${mapping_file}
+}
+trap finish EXIT
+
+MIRROR_ORIGIN_TESTS_OC_TOOL="${MIRROR_ORIGIN_TESTS_OC_TOOL:-oc}"
+MIRROR_ORIGIN_TESTS_CONTAINER_MGMT_CLI="${MIRROR_ORIGIN_TESTS_CONTAINER_MGMT_CLI:-docker}"
+
+MIRROR_ORIGIN_TESTS_IN_CONTAINER="${MIRROR_ORIGIN_TESTS_IN_CONTAINER:-true}"
+MIRROR_ORIGIN_TESTS_IMAGE="${MIRROR_ORIGIN_TESTS_IMAGE:-quay.io/openshift/origin-tests:$OCP_VERSION}"
```

Comment:
this one can be shared between the run and mirror
so they both use the same parameter name.